base/subarray.jl

```diff
@@ -527,7 +527,22 @@ function parentdims(s::SubArray)
 end
 
 ## Scalar indexing
-getindex(V::SubArray, I::Int...) = (@_inline_meta; checkbounds(V, I...); unsafe_getindex(V, I...))
```

Comment:
Yup, that's what I tried first, but this loads before `array.jl` in the sysimg, so the machinery to do that isn't available yet.